src/main/host/syscall/handler/time.rs

```diff
@@ -96,4 +97,105 @@ impl SyscallHandler {
 
         Ok(0.into())
     }
+
+    #[log_syscall(/* clock_id */ linux_api::time::linux___kernel_clockid_t,
+        /* flags */ std::ffi::c_int, /* request */ *const std::ffi::c_void,
```

Comment:
Optional - we could get nicer syscall logging by using the higher level types `ClockId` and `ClockNanosleepFlags` here (not in the signature itself). You would need to add a conversion and formatting rule for `ClockNanosleepFlags`  though. For an example see

```
main/host/syscall/type_formatting.rs:simple_debug_impl!(linux_api::time::ClockId);
```

and

```
lib/shadow-shim-helper-rs/src/syscall_types.rs:impl TryFrom<SysCallReg> for linux_api::time::ClockId {
```